content/md/en/docs/tutorials/smart-contracts/first-smart-contract.md

```diff
@@ -0,0 +1,475 @@
+---
+title: Prepare your first contract
+slug: /tutorials/get-started/smart-contract/
+description: 
+keywords:
+  - smart contracts
+  - erc20
+  - wasm
+difficulty: 2
+duration: 2.5 Hours
+relevantSkills:
+  - ink!
+  - Smart Contracts
+---
+
+As you learned in [Blockchain basics](/main-docs/fundamentals/blockchain-basics/) decentralized applications are most often written as **smart contracts**.
+Although Substrate is primarily a framework and toolkit for building custom blockchains, it can also provide a platform for smart contracts.
+This tutorial demonstrates how to build a basic smart contract to run on a Substrate-based chain. 
+In this tutorial, you'll explore using ink! as a programming language for writing Rust-based smart contracts.
+
+## Before you begin
+
+Before you begin, verify the following:
+
+* You have good internet connection and access to a shell terminal on your local computer.
+
+* You are generally familiar with software development and using command-line interfaces.
+
+* You are generally familiar with blockchains and smart contract platforms.
+
+* You have installed Rust and set up your development environment as described in [Install](/main-docs/install/).
+
+## Tutorial objectives
+
+By completing this tutorial, you will accomplish the following objectives:
+
+* Learn how to create a smart contract project.
+
+* Build and test a smart contract using the ink! smart contract language.
+
+* Deploy a smart contract on a local Substrate node.
+
+* Interact with a smart contract through a browser.
+
+## Update your Rust environment
+
+For this tutorial, you need to add some Rust source code to your Substrate development environment.
+
+To update your development environment:
+
+1. Open a terminal shell on your computer.
+
+1. Change to the root directory where you compiled the Substrate node template.
+
+1. Update your Rust environment by running the following command:
+    
+    ```bash
+    rustup component add rust-src --toolchain nightly
+    ```
+
+1. Verify that you have the WebAssembly target installed by running the following command:
+    
+    ```bash
+    rustup target add wasm32-unknown-unknown --toolchain nightly
+    ```
+
+    If the target is installed and up-to-date, the command displays output similar to the following:
+
+    ```bash
+    info: component 'rust-std' for target 'wasm32-unknown-unknown' is up to date
+    ```
+
+## Install the Substrate contracts node
```

Comment:
On `master` we currently have this note:

>You can [download a binary from our releases page](https://github.com/paritytech/substrate-contracts-node/releases) (Linux and Mac). Alternatively you can build the node by yourself:

Could you add something like this here? Installing the node takes a long time and there's no real need for it (except if someone has e.g. an exotic operating system), so we should prioritize linking the binary releases here.

Reply:
Oh that's cool. I didn't see that. Changing this section.